nominal/_utils/download_tools.py

```diff
@@ -0,0 +1,28 @@
+import pathlib
+from urllib.parse import unquote, urlparse
+
+import httpx
```

Comment:
why httpx instead of requests? you can stream with requests as well fyi

Reply:
fwiw i actually like httpx's interface a lot but used requests for pairity with the conjure clients